save-backend/src/main/kotlin/org/cqfn/save/backend/service/ExecutionService.kt

```diff
@@ -36,4 +40,18 @@ class ExecutionService(private val executionRepository: ExecutionRepository) {
             throw ResponseStatusException(HttpStatus.NOT_FOUND)
         }
     }
+
+    /**
+     * @param executionId id of execution
+     * @return execution dto based on id
+     */
+    fun getExecutionDto(executionId: Long): ExecutionDto? {
+        var executionDto: ExecutionDto? = null
+        executionRepository.findById(executionId).ifPresentOrElse({
+            executionDto = it.toDto()
+        }) {
+            log.error("Can't find execution by id = $executionId")
```

Comment:
You shouldn't silently ignore exception and return `null` to front, you should rather return `4xx` error